examples/input_output/systemSpeakExample/src/ofApp.cpp

```diff
@@ -31,17 +31,17 @@ void ofApp::threadedFunction() {
 
 		#ifdef TARGET_OSX
 			string cmd = "say -v "+voice+" "+words[step]+" ";   // create the command
-			system(cmd.c_str());
+			if(system(cmd.c_str())) {};
```

Comment:
Uh, this looks a bit odd ... should be able to stand alone, no? Does gcc print a warning for this line?

Reply:
Yes, unused return value.

Reply:
For code which part of examples, it could make sense to add something to use the return value. I'm just not sure an if-statement is the clearest way of expressing the intent. 

How about: 

`auto status = system(cmd.c_str());` ?

Reply:
Will lead to another gcc warning. Unused variable.

Reply:
What about **std::ignore = system(cmd.c_str());** ?

Looks idiomatic although it isn't really.

Reply:
**std::ignore** is C++11. Probably not what we want. I am not sure.

Note: (void) fn(...), static_cast, and so on do not work. I even used a template, but might be overkill:

```
template <typename T>
void UnusedReturnValue(T&& /*No name*/) { /*Empty*/ }
```
Then:
```
UnusedReturnValue(system(cmd.c_str()));
```

I am afraid we have to live with that warning in gcc6. Gcc6 should just accept **(void) system(...)** that would be easiest and also upward compatible.

Reply:
Hmm... probably best to just live with the "unused return value" warning - it's one of the most benign warnings anyway... 

Putting the "unused return values" aside  - if the `unsigned` were changed to `size_t`, this PR would be a good candidate to merge =)